components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/config/model/graph/js/JsAuthenticatedUser.java

```diff
@@ -37,10 +39,17 @@
  */
 public class JsAuthenticatedUser extends AbstractJSObjectWrapper<AuthenticatedUser> {
 
-    public JsAuthenticatedUser(AuthenticatedUser wrapped) {
+    private AuthenticationContext context;
```

Comment:
Whats the idea behind having the context within the wrapped user object?

Reply:
This is required to access step level claims and needed to be passed to JsClaims.
Since we'll be getting the context always, it should be fine to have it. WDYT?